client/src/components/AnswerModal/index.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import styles from './AnswerModal.css'
+
+class AnswerModal extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      question: '',
+      nickname: '',
+      email: '',
+      questionError: null,
+      nicknameError: null,
+      emailError: null
+    }
+    this.validate = this.validate.bind(this);
+    this.handleOnChange = this.handleOnChange.bind(this);
+    this.handleSubmitClick = this.handleSubmitClick.bind(this);
+  }
+
+  validate() {
+    { this.state.question.length < 1 ? this.setState({ questionError: 'Please enter a question' }) : this.setState({ questionError: null }) }
+    { this.state.nickname.length < 1 ? this.setState({ nicknameError: 'Please enter a nickname' }) : this.setState({ nicknameError: null }) }
+    { !this.state.email.includes('@') ? this.setState({ emailError: 'Please enter valid email' }) : this.setState({ emailError: null }) }
```

Comment:
You should probably group all your `setState()` into 1 

```suggestion
    const { question, nickname, email } = this.state;
    const questionError = question.length < 1 ? 'Please enter a question' : null;
    const nicknameError = nickname.length < 1 ? 'Please enter a nickname' : null;
    const emailError = email.includes('@') ? 'Please enter valid email' : null;
    this.setState({
      questionError,
      nicknameError,
      emailError,
    });
```

Reply:
destructured state.